apps/common-app/src/examples/RuntimeTests/RuntimeTestsExample.tsx

```diff
@@ -52,6 +52,31 @@ export default function RuntimeTestsExample() {
             require('./tests/utilities/relativeCoords.test');
           },
         },
+        {
+          testSuiteName: 'entering and exiting',
+          importTest: () => {
+            require('./tests/layoutAnimations/entering/enteringColors.test');
+            require('./tests/layoutAnimations/entering/predefinedEntering.test');
+            require('./tests/layoutAnimations/exiting/predefinedExiting.test');
+          },
+        },
+        {
+          testSuiteName: 'layout transitions',
+          importTest: () => {
+            describe('Compare layout transitions with **constant view size** with snapshots', () => {
+              require('./tests/layoutAnimations/layout/predefinedLayoutPosition.test');
+            });
+            describe('Compare Test layout transitions including view **size changes** with snapshots', () => {
+              require('./tests/layoutAnimations/layout/positionAndSize.test');
+            });
+          },
+        },
```

Comment:
I don't think it should be here, right?

Reply:
I was adding more sections in this PR, so I thought I can split existing ones in this PR too.

Reply:
I don't understand. If these test were added already, why weren't they imported in `RuntimeTestsExample` before?